packages/adapter-node/index.d.ts

```diff
@@ -19,6 +19,7 @@ interface AdapterOptions {
 			host?: string;
 		};
 	};
+	trustProxy?: boolean;
```

Comment:
this should probably be added to the README

Reply:
done